styles.css

```diff
@@ -0,0 +1,2153 @@
+    padding: 0;
+    gap: 8px;
+    width: 30%;
+    position: absolute;
+    height: 16.5%;
+  }
+
+  .tags {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 8px;
+    width: 16%;
+    position: absolute;
+    height: 15%;
+  }
+
+  .tag1 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 12px;
+    width: 53px;
+    height: 24px;
+    background: #ebebff;
+    border-radius: 8px;
+  }
+
+  .tag12 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 12px;
+    width: 28%;
+    height: 24px;
+    background: #ebebff;
+    border-radius: 8px;
+  }
+
+  #br {
+    display: none;
+  }
+
+  .tag2 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 12px;
+    width: 45px;
+    height: 24px;
+    background: #ebebff;
+    border-radius: 8px;
+  }
+
+  .tag3 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 12px;
+    width: 89px;
+    height: 24px;
+    background: #ebebff;
+    border-radius: 8px;
+  }
+
+  .html {
+    width: 29px;
+    height: 16px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .css {
+    width: 21px;
+    height: 16px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .ruby {
+    width: 100%;
+    height: 16px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .js {
+    width: 65px;
+    height: 16px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .pry-text {
+    width: 88%;
+    height: 76px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 13px;
+    line-height: 24px;
+    color: #344563;
+    margin-top: 0%;
+  }
+
+  .see-project {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 0;
+    position: relative;
+    top: 0%;
+    width: 130px;
+    height: 48px;
+    left: 0.8%;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    background-color: #fff;
+    cursor: pointer;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+  }
+
+  .see-project-l {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
```

Comment:
- [ ] At some breakpoint this button loses the right layout, I highly suggest removing the absolute position from the button so then the layout will be easily achieved by only providing some margins 👏👌

### Your Design
![image](https://user-images.githubusercontent.com/68030297/221351663-32073701-f5c6-48cf-98e4-e61a5ad36330.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/221351679-c91b738a-2d35-4424-9b04-a8b6493f5d1f.png)